lib/ansible/modules/net_tools/ipwcli_dns.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Christian Wollinger <cwollinger@web.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: ipwcli_dns
+
+short_description: Manage DNS Records for Ericsson IPWorks via ipwcli
+
+version_added: "2.10"
+
+description:
+    - "Manage DNS records for the Ericsson IPWorks DNS server. The module will use the ipwcli to deploy the DNS records."
+
+requirements:
+   - ipwcli (installed on Ericsson IPWorks)
+
+notes:
+    - To make the DNS record changes effective, you need to run 'update dnsserver' on the ipwcli.
```

Comment:
Wouldn't it make sense if the module does that as well?

Also:
```suggestion
    - To make the DNS record changes effective, you need to run C(update dnsserver) on the ipwcli.
```

Reply:
I would not recommend this, because the update dnsserver will take about 10 minutes in a productive environment. Therefore it would be useless for a loop on the module.